python-frontend/src/test/java/org/sonar/python/semantic/ProjectLevelSymbolTableTest.java

```diff
@@ -1125,6 +1125,18 @@ class Field(MetaField): ...
     assertThat(symbol.hasUnresolvedTypeHierarchy()).isTrue();
   }
 
+  @Test
+  void class_wth_metaclass() {
```

Comment:
It would probably make sense to have some other tests where the metaclass is local or unresolved. I know it's basically testing `typeFqn` again, but since it's not a fully valid implementation (cf above comment), we probably want to have it documented.